README.md

```diff
@@ -197,24 +160,10 @@ If you have a bug to report, reproduce it in an online code editor. For example,
 
 ## Missing features
 
-See the list of features that are still missing in the component:
-
-- [ ] render [`bindings`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#fixed-fields-19)
-- [ ] render [`tags`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#tagsObject)
-- [ ] render [`externalDocs`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#externalDocumentationObject)
-- [ ] render [`schema properties`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#properties) 
-- [ ] render [`correlationID`](https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#correlationIdObject)
+To see features we don't support, please check [issues](https://github.com/asyncapi/asyncapi-react/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement).
```

Comment:
not all enhancements are about adding missing features. In my opinion better just remove this section

Reply:
You're right.